README.md

```diff
@@ -64,7 +63,6 @@ This repository contains code that builds the following tools in `bin/`:
  * **wasm-opt**: Loads WebAssembly and runs Binaryen IR passes on it.
  * **asm2wasm**: An asm.js-to-WebAssembly compiler, using Emscripten's asm optimizer infrastructure. This is used by Emscripten in Binaryen mode when it uses Emscripten's fastcomp asm.js backend.
  * **wasm2asm**: A WebAssembly-to-asm.js compiler (still experimental).
- * **s2wasm**: A compiler from the `.s` format emitted by the new WebAssembly backend being developed in LLVM. This is used by Emscripten in Binaryen mode when it integrates with the new LLVM backend.
```

Comment:
while we update this list, maybe worth adding `wasm-emscripten-finalize` and `wasm-link-metadata`?